src/main/kotlin/com/jetbrains/snakecharm/inspections/SmkShadowSettingsInspection.kt

```diff
@@ -0,0 +1,37 @@
+package com.jetbrains.snakecharm.inspections
+
+import com.intellij.codeInspection.LocalInspectionToolSession
+import com.intellij.codeInspection.ProblemsHolder
+import com.jetbrains.python.psi.PyStringLiteralExpression
+import com.jetbrains.snakecharm.SnakemakeBundle
+import com.jetbrains.snakecharm.codeInsight.completion.ShadowSectionSettingsProvider
+import com.jetbrains.snakecharm.lang.SnakemakeLanguageDialect
+import com.jetbrains.snakecharm.lang.psi.SMKRuleParameterListStatement
+
+class SmkShadowSettingsInspection : SnakemakeInspection()  {
+    override fun buildVisitor(
+            holder: ProblemsHolder,
+            isOnTheFly: Boolean,
+            session: LocalInspectionToolSession
+    ) = object : SnakemakeInspectionVisitor(holder, session) {
+
+        override fun visitSMKRuleParameterListStatement(st: SMKRuleParameterListStatement) {
+            if (!SnakemakeLanguageDialect.isInsideSmkFile(st) ||
+                    st.name != SMKRuleParameterListStatement.SHADOW) {
+                return
+            }
+
+            val stringLiteral = st.argumentList?.arguments?.find {
+                it is PyStringLiteralExpression
```

Comment:
Does snakemake allow more that one arg here in arg list for "shadow" section? If not, than we could also highlight it as a problem

Reply:
Only one argument after **shadow** section is permitted. If you try to run this script:
```
rule NAME:
    shadow: "full", "shallow"
```
you will get an error: 
_shadow() takes 2 positional arguments but 3 were given_